daemon/SecretsImpl/secretsrequestprocessor.cpp

```diff
@@ -42,6 +42,23 @@ namespace {
                              ident.name()).toUtf8(),
                     QCryptographicHash::Sha512).toBase64());
     }
+
+    QString determineAuthPlugin(const QString &ownerApplicationId,
+                                const QString &callerApplicationId,
+                                bool callerIsPlatformApplication,
+                                const QString &authPluginName,
+                                const QString &interactionServiceAddress,
+                                bool autotestMode) {
+        const QString defaultPluginName = autotestMode
```

Comment:
In test mode, shouldn't we always just use the in-app plugin?

Reply:
No, some tests explicitly use the password auth plugin (e.g. the test which uses the example usb plugin, and triggers plugin unlocking, etc).